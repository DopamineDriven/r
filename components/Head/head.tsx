import { FC } from 'react';
import { DefaultSeo } from 'next-seo';
import config from '../../config/seo.json';
import NextHead from 'next/head';

const Head = () => {
	return (
		<>
			<DefaultSeo {...config} />
			<NextHead>
				<meta
					name='viewport'
					content='width=device-width, initial-scale=1'
				/>
				<meta property={'og:type'} content='website' />
				<meta
					property='og:url'
					content='https://r-weld.vercel.app'
				/>
				<meta property='og:title' content='Subreddit Search' />
				<meta
					property='og:image'
					content='https://og-image.now.sh/%2Fr%2F%3Fq%3Dreq.query.q.png?theme=dark&md=1&fontSize=100px&images=https%3A%2F%2Fassets.vercel.com%2Fimage%2Fupload%2Ffront%2Fassets%2Fdesign%2Fnextjs-white-logo.svg&images=https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2Fremojansen%2Flogo.ts%40master%2Fts.svg&images=https%3A%2F%2Fdev-to-uploads.s3.amazonaws.com%2Fuploads%2Farticles%2F9rqndpi9iisgalx7bp0z.png'
				/>
				{/* <link rel='manifest' href='' /> */}
			</NextHead>
		</>
	);
};

export default Head;
