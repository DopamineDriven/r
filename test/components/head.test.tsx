import Head from '../../components/Head/head';
import { mocked } from 'ts-jest/utils';
import { render } from '@testing-library/react';

describe('Head MetaData', () => {
	it('matches snapshot', () => {
		const { container } = render(<Head />);
		expect(
			container.getElementsByTagName('meta').namedItem('viewport')
		);
		expect(container).toMatchSnapshot();
	});
});
