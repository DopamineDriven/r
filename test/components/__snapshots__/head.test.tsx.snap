// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head MetaData matches snapshot 1`] = `<div />`;
